storybook/mattermost_storybook.ts

```diff
@@ -2,7 +2,9 @@
 // See LICENSE.txt for license information.
 import {Navigation} from 'react-native-navigation';
 import {goToScreen} from '@actions/navigation';
+import {withReduxProvider} from '@screens';
 
 import DevMenu from 'react-native-dev-menu';
 DevMenu.addItem('StoryBook', () => goToScreen('StoryBook', 'StoryBook'));
-Navigation.registerComponent('StoryBook', () => require('../storybook').default, () => require('../storybook').default);
+
+Navigation.registerComponent('StoryBook', () => withReduxProvider(require('../storybook').default), () => require('../storybook').default);
```

Comment:
Hooking storybook up to our state context so that I can access user data (profile images in this case) using userIDs